products/urls.py

```diff
@@ -1,10 +1,14 @@
 from django.urls import path
 from django.conf.urls.static import static
 from django.conf import settings
-from .views import home_page
-
+from .views import home_page, category_data, \
+    subcategory_data, product_data, all_products
 
 urlpatterns = [
     path('', home_page, name='homepage'),
     path('profile/', home_page, name='admin-profile'),
```

Comment:
@Yshah112 give proper naming convensions to urls. here you have added `products` in 2 urls.
   ```
 path('product/<int:pk>/', product_data, name='products'),
 path('products/', all_products, name='all-products')
```
cc: @lalitvasoya

Reply:
done